### PR TITLE
Userguide basics 4 6

### PR DESCRIPTION
**Pull request by hirunatan**

### CODE REVIEW

File: user-guide/objects/index.njk [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  <li><strong>Text decoration:</strong> none, underline, strikethrough.</li>
+</ol>
+
+
+<h3 id="paths">Paths (bezier)</h3>
+<p>A path is composed by two or more nodes that will form a vector shape. To draw a new path you have to activate the text path by clicking on the icon at the toolbar or pressing <kbd>P</kbd>. Then you have two ways to create a text shape:</p>
+<ol>
+  <li><strong>Click</strong> to create a new corner node.</li>
+  <li><strong>Click and drag</strong> to create a curved node.</li>
+</ol>
```
Comment (hirunatan) [inline user-guide/objects/index.njk:61]: Perhaps it's interesting to explain how to finish a path, leaving it open or closed.

File: user-guide/workspace-basics/index.njk [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+<h1 id="workspace-basics">Workspace basics</h1>
+<p class="main-paragraph">The Workspace is where you actually create your designs. You have an infinite canvas where you can work directly but you also have the ability to create and work inside artboards that will help you to create pages and exportation units.</p>
+
+<h3 id="viewport">The viewport</h3>
+<p>Surrounded by panels, header and toolbars, in the middle of the workspace, you can find the viewport. The viewport is the design area of a file page. It is practically infinite. If what you need is a frame with specific, limited dimensions, you can create an artboard.</p>
+<p><img src="/img/viewport.png" alt="the viewport" /></p>
+<h4>Navigate the viewport</h4>
+<p>Press Tab (your cursor) while moving your mouse to navigate the viewport. If you are using a trackpad you can do two finger scrolling.</p>
```
Comment (hirunatan) [inline user-guide/workspace-basics/index.njk:12]: This is by pressing space, not Tab, isn't it? And what does "your cursor" mean?

File: user-guide/layer-basics/index.njk [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+
+
+<h3 id="rotating-layers">Rotating layers</h3>
+<p>To rotate selected layers you can use the handlers at the edges of the selection box. Make sure the cursor is in rotation mode. If you hold <kbd>Ctrl/⌘</kbd> while rotation the angle will change in 45 degree increments. You can also find this option at the design panel.</p>
+<p><img src="/img/layers-rotating.gif" alt="layers" /></p>
+
+
+<h3 id="flipping-layers">Flipping layers</h3>
+<p>To flip a layer <strong>horizontally</strong> press <kbd>Ctrl/⌘</kbd> + <kbd>H</kbd>. To flip a layer <strong>vertically</strong> press <kbd>Ctrl/⌘</kbd> + <kbd>V</kbd>. You can also find these actions at the layers menu.</p>
```
Comment (hirunatan) [inline user-guide/layer-basics/index.njk:98]: This is with Shift-h and Shift-v (Ctrl-h is for showing the history panel)

File: user-guide/workspace-basics/index.njk (hunk 2 above)
Comment (myfunnyandy) [reply user-guide/workspace-basics/index.njk]: dunnow who wrote that :P

**Pull request merged**